client/login/wp-login/index.jsx

```diff
@@ -124,29 +126,43 @@ class Login extends React.Component {
 			onClick={ this.goBack }>
 				<Gridicon icon="arrow-left" size={ 18 } /> { this.props.translate( 'Return' ) }
 			</a>;
```

Comment:
I believe the return link should be the last link displayed in the footer. First, because that's where it is located on the `Verification Code` page. And second, because that's probably the less useful action from this list (I mean the user most certainly already knows how to navigate back and forth with their browser). For information, that's where it is displayed on the current [`Login` page](https://wordpress.com/wp-login.php):

<img width="312" alt="screenshot" src="https://cloud.githubusercontent.com/assets/594356/26033689/98846312-38b0-11e7-9495-fa224ba63066.png">